keps/sig-multicluster/3335-statefulset-slice/kep.yaml

```diff
@@ -0,0 +1,39 @@
+title: StatefulSet Slice
+kep-number: 3335
+authors:
+  - "@pwschuurman"
+owning-sig: sig-multicluster
+participating-sigs:
+  - sig-apps
+  - sig-storage
+status: provisional
+creation-date: 2022-06-02
+reviewers:
+  - TBD
+approvers:
+  - TBD
```

Comment:
IIUC based on SIG-MC comments today, sounds like this needs at least a SIG-MC chair and a SIG-Apps chair

Reply:
The above lists should be filled in, otherwise we don't know whom do you expect to approve this.